src/ui/screens/ScreenGetHelp.qml

```diff
@@ -4,67 +4,66 @@
 
 import QtQuick 2.5
 import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.15
 
 import Mozilla.VPN 1.0
 import components 0.1
 
 Item {
-    property int unlockCounter: 0
-
     Rectangle {
         anchors.fill: parent
         color: window.color
     }
 
-    VPNMenu {
-        id: menu
-        objectName: "getHelpBack"
-        _menuOnBackClicked: () => VPNNavigator.requestPreviousScreen()
-        _menuIconButtonSource: getHelpStackView.depth === 1 ? "qrc:/nebula/resources/close-dark.svg" : "qrc:/nebula/resources/back.svg"
-    }
+    ColumnLayout {
+        anchors.fill: parent
 
-    VPNMouseArea {
-        anchors.fill: menu
-        hoverEnabled: getHelpStackView.depth === 1
-        cursorShape: Qt.ArrowCursor
-        onMouseAreaClicked: function() {
-            if (unlockCounter >= 5) {
-                unlockCounter = 0
-                VPNSettings.developerUnlock = true
-            }
-            else if (!VPNSettings.developerUnlock) {
-                unlockTimeout.restart()
-                unlockCounter = unlockCounter + 1
-            }
-        }
-    }
+        spacing: 0
 
-    Timer {
-        id: unlockTimeout
-        repeat: false
-        running: false
-        interval: 10000
-        onTriggered: unlockCounter = 0
-    }
+        VPNMenu {
+            id: menu
+
+            property int unlockCounter: 0
 
-    VPNStackView {
-        id: getHelpStackView
+            objectName: "getHelpBack"
+            _menuOnBackClicked: () => VPNNavigator.requestPreviousScreen()
+            _menuIconButtonSource: getHelpStackView.depth === 1 ? "qrc:/nebula/resources/close-dark.svg" : "qrc:/nebula/resources/back.svg"
+            titleClicked: () => {
+                            if (unlockCounter >= 5) {
```

Comment:
Nit: I think it would be nice to have threshold values — in this case the amount of clicks needed in order “unlock” the developer options — in `constants`. Wdyt?

Reply:
We could consider exposing `constants.cpp` to the QML engine, or creating a frontend only constants.js file (similar to themes.js)

Reply:
Thinking about it again: It might be nice to have, but let’s do it in case there is a need for that in other places as well. Doing this for the developer options as an isolated case might be not the best incentive.